python/model.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+
+import argparse
+from lib.database import *
+from lib.tests import *
+from lib.colors import colors
+import random
+
+
+def main():
+    parser = argparse.ArgumentParser(description="Database setup and testing script")
+    parser.add_argument(
+        "-q", "--quick", action="store_true", help="Perform quick setup"
+    )
+    parser.add_argument(
+        "-b",
+        "--batch-size",
+        type=int,
+        default=1e5,
+        help="Batch size for populating fact tables",
+    )
+    parser.add_argument(
+        "-m",
+        "--mode",
+        choices=["build", "test"],
+        default="build",
+        help="Mode: build or test",
+    )
+    parser.add_argument(
+        "-s",
+        "--step",
+        action="store_true",
+        help="Engage input() calls in the test routine for step-by-step execution",
+    )
+    args = parser.parse_args()
+
+    BE_QUICK_ABOUT_IT = args.quick
+    BATCH_SIZE = args.batch_size
+    BUILD = args.mode == "build"
+    TEST = args.mode == "test"
+    STEP = args.step
+
+    db = get_db_handle()
+
+    # fmt: off
+    if BUILD is True:
+        disconnect_other_users(db)
+        drop_schemata_except(db)
+        drop_public_entities(db)
+        ensure_extensions_exists(db)
+        pull_down_locations(db)
+        create_schemata(db)
+        create_lookup_tables(db)
+        artifically_descync_sequences_from_cris_data(db, ["cris_lookup.road_types_sequence", "cris_lookup.unit_types_sequence"])
+        populate_lookup_tables(db)
+        refresh_materialized_views(db)
+        create_fact_tables(db)
+        create_cris_facts_functions(db)
+        create_cris_facts_triggers(db)
+        create_temporal_tracking(db)
+        artifically_descync_sequences_from_cris_data(db, ["cris_facts.crashes_id_seq", "cris_facts.units_id_seq","visionzero_facts.crashes_id_seq", "visionzero_facts.units_id_seq"])
+        create_lookup_table_substitution_triggers(db)
+        populate_fact_tables(db, BE_QUICK_ABOUT_IT, batch_size=BATCH_SIZE)
+        create_unifying_fact_views(db)
+
```

Comment:
After this point, there are no mentions of triggers in the code. No edits, creations or deletions of triggers at all. They are all done in general and in the setup of the system.